tests/testthat/test-censor-probs.R

```diff
@@ -0,0 +1,65 @@
+
+test_that("reverse Kaplan-Meier curves", {
+  skip_if_not_installed("prodlim")
+  skip_if_not_installed("survival")
```

Comment:
this should be covered by the next line since `survival` is a strong dependency of `censored`